internal/gps/pkgtree/pkgtree.go

```diff
@@ -189,7 +199,15 @@ func ListPackages(fileRoot, importRoot string) (PackageTree, error) {
 		return PackageTree{}, err
 	}
 
-	return ptree, nil
+	return ptree, checkError(ptree)
+}
+
+func checkError(ptree PackageTree) error {
```

Comment:
i could be persuaded to add a method to `PackageTree` that takes over some of the `checkErrors()` responsibility (depending on what it actually does, ofc)